packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -40,62 +100,225 @@ export class SankeyChartBase extends React.Component<
       theme: theme!,
       width: this.state.containerWidth,
       height: this.state.containerHeight,
-      pathColor: pathColor,
+      pathColor,
       className,
     });
-    const margin = { top: 10, right: 0, bottom: 10, left: 0 };
-    const width = this.state.containerWidth - margin.left - margin.right;
-    const height =
-      this.state.containerHeight - margin.top - margin.bottom > 0
-        ? this.state.containerHeight - margin.top - margin.bottom
-        : 0;
+    const margin = { top: 36, right: 48, bottom: 32, left: 48 };
+    const width = this.state.containerWidth - margin.right;
+    const height = this.state.containerHeight - margin.bottom > 0 ? this.state.containerHeight - margin.bottom : 0;
```

Comment:
when can height be possibly 0

Reply:
This is just a check so that height can never be negative.